announcements/1.14.0.md

```diff
@@ -0,0 +1,87 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.14.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Charmhub migration
+
+All charms part of Anbox Cloud and used 3rd party charms now coming from [Charmhub](https://charmhub.io) instead of the old Juju Charm Store.
```

Comment:
```suggestion
All charms that are part of Anbox Cloud and the used 3rd-party charms now come from [Charmhub](https://charmhub.io) instead of the old Juju charm store.
```